src/perf/simulations/monitorAnalysis.scala

```diff
@@ -0,0 +1,70 @@
+package default
+
+import java.io.File
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+case class SubmissionStatus(
+	submissionId: String,
+	submissionStatus: String,
+	workflowStatuses: Vector[String]
```

Comment:
`findAll.saveAs` below uses `Vector`s, and I have no compelling reason not to do so as well